Tests/Router/DPLDeepLinkRouterSpec.m

```diff
@@ -139,47 +140,41 @@
         });
     });
     
-    it(@"matches less specfic routes first when they are registered first", ^{
+    it(@"continues looking for a match which will handle the route, even if the first match chooses not to handle it", ^{
         waitUntil(^(DoneCallback done) {
-            router[@"/say/:word"] = ^(DPLDeepLink *deepLink) {
-                expect(deepLink.routeParameters[@"word"]).to.equal(@"hello");
-            };
-            
-            router[@"/say/hello"] = ^{
-                XCTFail(@"The wrong route was matched.");
+            [router registerHandlerClass:[DPLWontHandleRouteHandler class] forRoute:@"/say/hello"];
+
+            __block BOOL didMatchOnSecondRegistration = NO;
+            router[@"/say/:word"] = ^{
+                didMatchOnSecondRegistration = YES;
             };
             
             BOOL isHandled = [router handleURL:url withCompletion:^(BOOL handled, NSError *error) {
                 expect(handled).to.beTruthy();
                 expect(error).to.beNil();
                 done();
             }];
+            expect(didMatchOnSecondRegistration).to.beTruthy();
             expect(isHandled).to.beTruthy();
         });
     });
     
-    it(@"produces an error when a URL has no matching route", ^{
-        waitUntil(^(DoneCallback done) {
-            BOOL isHandled = [router handleURL:url withCompletion:^(BOOL handled, NSError *error) {
-                expect(handled).to.beFalsy();
-                expect(error.code).to.equal(DPLRouteNotFoundError);
-                done();
-            }];
-            expect(isHandled).to.beFalsy();
-        });
-    });
-    
-    it(@"produces an error when a route handler does not specify a target view controller", ^{
```

Comment:
Looks like this test was removed too. It's still invalid to not specify a target view controller. Why is this test removed?